apps/backend/assets/README.md

```
@@ -0,0 +1,49 @@
+- **Next Review Date**: February 2025
+
+**PRODUCTION DEPLOYMENT CHECKLIST**:
+- [ ] TAR form license verified and documented
+- [ ] Legal team review completed
+- [ ] Template version verified against current TAR standards
+- [ ] Proper attribution included in generated PDFs
+- [ ] Access controls configured (only authorized users can generate)
+
+**ALTERNATIVE SOLUTIONS**:
+If you do not have TAR authorization:
+1. Obtain TAR membership/license at https://www.texasrealestate.com
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)